### PR TITLE
Psp-1689

### PR DESCRIPTION
**Pull request by devinleighsmith**
tenant page for lease and licenses.
re-org some files based on category.

### CODE REVIEW

File: backend/dal/Migrations/20211013173510_v0.2.12.1.cs [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+using Pims.Dal.Helpers.Migrations;
+using System.Diagnostics.CodeAnalysis;
+
+namespace Pims.Dal.Migrations
+{
+    public partial class v02121 : SeedMigration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            PreUp(migrationBuilder);
```
Comment (devinleighsmith) [inline backend/dal/Migrations/20211013173510_v0.2.12.1.cs:11]: need a migration to handle missing seed values

File: frontend/src/features/leases/detail/LeasePages/details/DetailTermInformationBox.tsx [hunk 2]
```diff
@@ -48,7 +47,7 @@ const StyledDate = styled.div`
 
 const StyledTermInformationBox = styled.div`
   width: 25.5rem;
-  background-color: ${props => props.theme.css.primaryColor};
+  background: rgba(0, 33, 66, 0.73);
```
Comment (asanchezr) [inline frontend/src/features/leases/detail/LeasePages/details/DetailTermInformationBox.tsx:50]: This is okay for now but would love to see it added to the **colors.scss** file so we can re-use it when needed

File: frontend/src/utils/formUtils.ts [hunk 3]
```diff
@@ -7,3 +7,23 @@
...
+ * The phoneFormatter is used to format the specified phone number value
+ * @param {string} phoneNumber This is the target phone number to be formatted
+ */
+export const phoneFormatter = (phoneNumber?: string) => {
+  if (!!phoneNumber) {
+    let result = phoneNumber;
+    const regex =
+      phoneNumber.length === 10
+        ? /(\d\d\d)[\s-]?(\d\d\d)[\s-]?(\d\d\d\d)/
+        : /\d(\d\d\d)[\s-]?(\d\d\d)[\s-]?(\d\d\d\d)/;
```
Comment (asanchezr) [inline frontend/src/utils/formUtils.ts:21]: If we are going to support country codes in the phone number, this regex only allows for single digit country codes., e.g. USA/CAN (1). Other countries, say UK (44) would not be matched here.

Not sure it's worth supporting phone numbers outside of CAN but worth mentioning here.

https://countrycode.org/

File: frontend/src/features/leases/detail/LeasePages/details/DetailTermInformationBox.tsx (hunk 2 above)
Comment (devinleighsmith) [reply frontend/src/features/leases/detail/LeasePages/details/DetailTermInformationBox.tsx:50]: This is just the primary color with an alpha filter applied, I'm not convinced that deserves its own entry.

**Pull request merged**